uiowa/uiowa
# Iowa JPEC article migration

Resolves #5059 

Extra considerations for this migration:

- Only articles from 2019 on
- Include tags
- Skip previously migrated articles (12 in total)
- Create redirects
- Use the extra date field on the existing site to alter the "created" date on the new to match "publication" dates





# How to test


Sync www.iowajpec.org locally in dev desktop from uiowa702 (db name is `www_iowajpec_org`)
`git checkout iowajpec-migration && git pull`
`blt ds --site=iowajpec.org`
`drush @org-iowajpec.local en iowajpec_migrate -y && drush @org-iowajpec.local uli /admin/config/sitenow/sitenow-migrate`
Complete the form following the help text
`drush @org-iowajpec.local ms`
`drush @org-iowajpec.local mim --all`
Check there are no errors, check migrated content looks alright and that the above considerations are accounted for (such as publication dates match old site's, didn't duplicate manually moved articles, etc.)
